jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -2,19 +2,32 @@
 set -e
 
 # Sitespeed job script
+# Assumes that the required environment variables are set
+# either by the jenkins job or in the shell
 # Assumes the following environment variables are set
 #
-# TEST_URL
-# EDX_USER
-# EDX_PASS
-# USE_BASIC_AUTH
-# AUTH_USER (optional, required if USE_BASIC_AUTH)
-# AUTH_PASS (optional, required if USE_BASIC_AUTH)
-# NUMBER_OF_TIMES
-# CONNECTION
-# SITESPEED_BROWSER
+# EDX_USER - Registered edX user for logging in and creating the session cookie file
+# EDX_PASS - Password for the registered edX user
+# USE_BASIC_AUTH - Is the site secured with basic auth (e.g. sandboxes are)
+# AUTH_USER (optional, required if USE_BASIC_AUTH) - Basic auth username
+# AUTH_PASS (optional, required if USE_BASIC_AUTH) - Basic auth password
+# NUMBER_OF_TIMES - The number of times to test each URL when fetching timing metrics
+# CONNECTION - Limit the speed by simulating connection types
+# SITESPEED_BROWSER - Which browser to use to collect timing data
```

Comment:
Bonus: Should we call out what those browsers are (so no one messes up syntax by saying Firefox vs firefox, etc?)